### PR TITLE
PRODX-20521 Update handleProtocolActivateCluster

### PR DESCRIPTION
**Pull request by bnalisnikovskiy**
Changed logic at `renderer.tsx` related to handleProtocolActivateCluster
Instead of redirect to GlobalPage (SyncView) we navigate to the cluster

### CODE REVIEW

**Comment by stefcameron** [comment]
👀

File: src/renderer/renderer.tsx [hunk 1]
```diff
@@ -157,17 +161,28 @@ export default class ExtensionRenderer extends LensExtension {
...
+      // During new design implementation we find a proper place fot this
+      this.navigate(ROUTE_GLOBAL_PAGE);
+      dispatchExtEvent({
+        type: EXT_EXTERNAL_ERROR_SHOW,
+        data: {
+          error: strings.renderer.clusterActions.error.clusterNotFound(
+            `${namespace}/${clusterName}`
+          ),
+        },
+      });
```
Comment (stefcameron) [inline src/renderer/renderer.tsx:184]: Should just be `Renderer.Components.Notifications.error(...)`. That will show a notification "toast" message in the upper-right corner of the Lens UI.

File: src/renderer/components/GlobalPage/SyncView.js [hunk 2]
```diff
@@ -250,45 +248,22 @@ export const SyncView = function () {
...
+  // so probably we remove this part and show only logger
+  // at renderer.tsx. When refactoring will be finished
+  // we return this(or similar) notification logic.
+  const handleExternalErrorEvent = useCallback(function (event) {
+    const results = rtv.check({ event }, { event: extExternalErrorShowTs });
+    if (results.valid && event?.data?.error) {
+      setActiveEventType(EXT_EXTERNAL_ERROR_SHOW);
+      setExtEventError(event.data.error);
+    }
+  }, []);
```
Comment (stefcameron) [inline src/renderer/components/GlobalPage/SyncView.js:266]: This shouldn't be necessary as we'll just show the error notification from `renderer.tsx` (see my other comment there).

**Review by stefcameron — CHANGES_REQUESTED** [review]
Nice clean-up work! I don't think we need the `EXT_EXTERNAL_ERROR_SHOW` event, though...

File: src/renderer/renderer.tsx (hunk 1 above)
Comment (bnalisnikovskiy) [reply src/renderer/renderer.tsx]: Thanks. I was looking something like this. Much better now!

File: src/renderer/components/GlobalPage/SyncView.js (hunk 2 above)
Comment (bnalisnikovskiy) [reply src/renderer/components/GlobalPage/SyncView.js]: Fixed

**Review by stefcameron — APPROVED** [review]
Looks good, thanks! I tested it locally, works in success and error cases. 👌

**Pull request merged**